src/dpms/bigQueryWidget.tsx

```diff
@@ -323,83 +370,91 @@ const BigQueryComponent = ({
       ) : (
         <div style={{ width: '29px' }}></div>
       );
-      if (searchTerm) {
-        const arrowIcon = hasChildren ? (
-          node.isOpen ? (
-            <>
-              <div
-                role="treeitem"
-                className="caret-icon right"
-                onClick={handleIconClick}
-              >
-                <iconDownArrow.react
-                  tag="div"
-                  className="icon-white logo-alignment-style"
-                />
-              </div>
-            </>
-          ) : (
-            <div
-              role="treeitem"
-              className="caret-icon down"
-              onClick={handleIconClick}
-            >
-              <iconRightArrow.react
-                tag="div"
-                className="icon-white logo-alignment-style"
-              />
-            </div>
-          )
-        ) : (
-          <div style={{ width: '29px' }}></div>
-        );
-        if (depth === 1) {
-          return (
-            <>
-              {arrowIcon}
-              <div role="img" className="db-icon" onClick={handleIconClick}>
-                <iconBigQueryProject.react
-                  tag="div"
-                  className="icon-white logo-alignment-style"
-                />
-              </div>
-            </>
-          );
-        } else if (depth === 2) {
-          return (
-            <>
-              {arrowIcon}
-              <div role="img" className="db-icon" onClick={handleIconClick}>
-                <iconDataset.react
-                  tag="div"
-                  className="icon-white logo-alignment-style"
-                />
-              </div>
-            </>
-          );
-        } else if (depth === 3) {
-          return (
-            <>
-              {arrowIcon}
-              <div role="img" className="table-icon" onClick={handleIconClick}>
-                <iconTable.react
-                  tag="div"
-                  className="icon-white logo-alignment-style"
-                />
-              </div>
-            </>
-          );
-        }
+      // if (searchTerm) {
```

Comment:
Did we intend to comment all of this out?

Reply:
As we did not have the api, we had commented the code. Now it is all implemented.